src/telliot_feeds/reporters/tips/listener/funded_feeds_filter.py

```diff
@@ -104,6 +104,16 @@ async def price_change(self, query_data: bytes, value_before: bytes) -> Optional
                 val = getattr(query, param)
                 setattr(datafeed.source, param, val)
 
+        # if no value before short circuit and include feed to be suggested
```

Comment:
It's a little unclear to me why returning `10000` when `_get_price_change(previous_val=0, current_val=0))` "includes a feed to be suggested".

This portion of the comment is a little redundant: `if no value before short circuit`, as it's easy to see that it's returning `if not value_before`.

Reply:
10000 represents 100 percent change, will update comment.